src/main/webapp/query-control.js

```diff
@@ -17,6 +17,8 @@ function registerServiceWorker() {
 function clearPreviousResult() {
   document.getElementById('map-title').innerText = '';
   document.getElementById('data-table').innerHTML = '';
+  document.getElementById('colors').style.display = 'none';
+  document.getElementById('setColor').value = "#0000ff";
```

Comment:
How about #0071bd instead? That's the color that the buttons are

Reply:
Fixed